ci-operator/jobs/openshift/origin-metrics/openshift-origin-metrics-release-3.10-postsubmits.yaml

```diff
@@ -1,8 +1,34 @@
 postsubmits:
   openshift/origin-metrics:
-  - agent: jenkins
```

Comment:
Just to be sure, are you removing the jenkins-backed job intentionally?

Reply:
yes, as far as I saw Jenkins job was not working for 3.10. so my intention with this PR is replacing the Jenkins job to build the images for metrics 3.10 with this.

Does that make sense?

Reply:
Yeah, it totally makes sense, just wanted to make sure it is intentional (for example we might have a weird bug in the generator which is not supposed to touch jenkins jobs).